### ISSUE TITLE
SpringDiExtensionSupportReactiveEndpointTest.cancelableReactiveEndpoint_clientCancel_serverUnsubscribeCallBackInvoked is flaky

### ISSUE THREAD

**Issue by mcollovati** [mcollovati/quarkus-hilla#431]
The test fails randomly during validation

```
Error:  Failures: 
Error:    SpringDiExtensionSupportReactiveEndpointTest>AbstractReactiveEndpointTest.cancelableReactiveEndpoint_clientCancel_serverUnsubscribeCallBackInvoked:69->AbstractReactiveEndpointTest.assertThatConnectionHasBeenClosed:118->AbstractReactiveEndpointTest.lambda$assertThatConnectionHasBeenClosed$0:119 
Expecting actual:
  "X"
to start with:
  "CLOSED: "

[INFO] 
Error:  Tests run: 113, Failures: 1, Errors: 0, Skipped: 0
```

**Comment by mcollovati** [comment]
The failure happens because Flow enabled heartbeat in Atmosphere, so the client may receive its `X` message.
The 'X' message can be skipped in the test HillaClient, as already done for empty messages

**Comment by mcollovati** [comment]
It should be fixed by #429 429
